examples/optional-type/optional_type.bal

```diff
@@ -1,16 +1,15 @@
 import ballerina/io;
 
-//This function optionally returns a `string` value.
-function getValue(string key) returns string|() {
+//This function optionally returns a `string` value. `?` here is equivalent to `|()`.
```

Comment:
Shall we rephrase the last bit to something like this: "`string?` is equivalent to `string|()`."?

Reply:
```suggestion
//This function optionally returns a `string` value. `string?` is equivalent to `string|()`.
```